### ISSUE TITLE
UI: better organize website layout

### ISSUE THREAD

**Issue by Aradon267** [magshimim-next/cv-next#138]

**Comment by TheGreenOak** [comment]
@Aradon267 tag me here when you finish #136

**Comment by Aradon267** [comment]
Ok
please note that this might be related to #141 and #172 but until #136 is finished(should be soon) if you want you can "sketch" an idea for #145

**Comment by TheGreenOak** [comment]
i dont think that #145 is entirelly related to this, its a different part of the website.
#172 is not necessary because #141 removes the feed entirelly

**Comment by Aradon267** [comment]
Correct about #172 but see inside " #141 does remove the navbar links so only the theme and logo bits"

**Comment by Aradon267** [comment]
@TheGreenOak done

**Comment by Aradon267** [comment]
Because we now have a dedicated About page we can leave the text there
We can leave the Login and Signout because now we have checks and we present based on need
The home and feed buttons can be removed tho
